<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="quit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QLabel" name="path_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>175</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Output path:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="path">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>170</y>
     <width>235</width>
     <height>30</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="interval">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>65</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>450</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Start capture</string>
   </property>
  </widget>
  <widget class="QLabel" name="interval_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture interval [s]:</string>
   </property>
  </widget>
  <widget class="QLabel" name="repeats_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of captures:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeats">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browse">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>380</width>
     <height>210</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="mag">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="scalebar">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>15</y>
     <width>180</width>
     <height>20</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Insert scalebar                      </string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mag_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Magnification:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="col_black">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>55</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="col_white">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>85</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>White</string>
   </property>
  </widget>
  <widget class="QLabel" name="col_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale bar color:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="posi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToMinimumContentsLength</enum>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="posi_label">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>125</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale bar position:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
